openj9.test.daa/src/test.daa/net/openj9/test/arithmetics/TestValidityChecking.java

```diff
@@ -748,6 +749,490 @@ public void testEmptyArrayForCheckPacked()
 
     }
 
+    @Test
+    public void testFailingArguments()
+    {
+        final byte[] a = new byte[] {(byte) 0x40, (byte) 0xF9, (byte) 0xF1, (byte) 0xF2, (byte) 0xC3};
+        int result = ExternalDecimal.checkExternalDecimal(a, 0, 5, DecimalData.EBCDIC_SIGN_EMBEDDED_TRAILING, 5);
+        Assert.assertEquals("Precision == bytesWithSpaces should result in return code of 3", 3, result);
+
+        result = ExternalDecimal.checkExternalDecimal(a, 0, 5, DecimalData.EBCDIC_SIGN_EMBEDDED_TRAILING, -5);
+        Assert.assertEquals("bytesWithSpaces < 0 should result in return code of 3", 3, result);
+
+        result = ExternalDecimal.checkExternalDecimal(a, 2, 5, DecimalData.EBCDIC_SIGN_EMBEDDED_TRAILING, 1);
+        Assert.assertEquals("bytesWithSpaces < 0 should result in return code of 3", 2, result);
```

Comment:
We expect this test case to return 2 or 3? Comment and expected value does not match here.